### PR TITLE
ROC-50 Add/Remove user from band (also related to ROC-49)

### PR DESCRIPTION
**Pull request by Ablosch2**
This is where I put buttons in the MatchProfileForm for adding and removing members to bands. I thought it was to go here.

### CODE REVIEW

File: src/components/MatchProfile/MatchProfileForm.js [hunk 1]
```diff
@@ -2,7 +2,25 @@ import Grid from '@mui/material/Grid';
 import Stack from '@mui/material/Stack';
 import AllChipsArray from "./AllChipsArray";
 import UserTypeButtons from "./UserTypeButtons";
+import { Button, ButtonGroup } from '@mui/material';
 
+const add_user_band_func = (user_id) => {
```
Comment (squaill) [inline src/components/MatchProfile/MatchProfileForm.js]: use camelCase in JS

File: src/components/UserProfile/PersonalDetailsForm.js [hunk 2]
```diff
@@ -26,6 +26,11 @@ export default function PersonalDetailsForm(props) {
         setGender(event.target.value);
     };
 
+    const [isActive, setIsActive] = useState(props?.user?.is_active || false);
```
Comment (squaill) [inline src/components/UserProfile/PersonalDetailsForm.js]: this code isn't merged. Did you work on the same branch? Always branch off of development

**Review by squaill — CHANGES_REQUESTED** [review]

File: src/components/MatchProfile/MatchProfileForm.js (hunk 1 above)
Comment (Ablosch2) [reply src/components/MatchProfile/MatchProfileForm.js]: What is camelCase?

File: src/components/UserProfile/PersonalDetailsForm.js (hunk 2 above)
Comment (Ablosch2) [reply src/components/UserProfile/PersonalDetailsForm.js]: I think maybe it is from the branch I made at first for setting IsActive. I think that needs to be merged first before this one.

File: src/components/MatchProfile/MatchProfileForm.js (hunk 1 above)
Comment (Ablosch2) [reply src/components/MatchProfile/MatchProfileForm.js]: Nevermind I changed it. It's like no spaces and underscore and capitalize every first letter of the words. See it now.

File: src/components/UserProfile/PersonalDetailsForm.js (hunk 2 above)
Comment (squaill) [reply src/components/UserProfile/PersonalDetailsForm.js]: you branched off of your other feature branch. You should create a new branch off of master and move the changes relevant to this work over

File: src/components/MatchProfile/MatchProfileForm.js (hunk 1 above)
Comment (squaill) [reply src/components/MatchProfile/MatchProfileForm.js]: almost. You used pascal case. The first letter of the first word should be lower case

File: src/components/UserProfile/PersonalDetailsForm.js (hunk 2 above)
Comment (Ablosch2) [reply src/components/UserProfile/PersonalDetailsForm.js]: I made a new branch and pushed it. Is this one good?

**Review by squaill — CHANGES_REQUESTED** [review]
I have some logistical concerns that we need to work through. We probably won't get to merging this today

**Pull request closed without merging**